core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -1388,6 +1490,31 @@ export class QueryDetail extends React.Component {
                                         </td>
                                         }
                                     </tr>
+                                    <tr>
+                                        <td className="info-title">
+                                            Blocked Time
+                                        </td>
+                                        <td className="info-text">
+                                            {query.queryStats.totalBlockedTime}
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td className="info-title">
+                                            Tasks
```

Comment:
I'm not sure that number of tasks at query level is meaningful, maybe it's useful info at stage level